### PR TITLE
backup install_packages.sh before installing

### PR DESCRIPTION
**Pull request by layered-esell**
This is in regards to https://github.com/spinnaker/spinnaker/issues/625 and should help with the issue I had when I updated the ubuntu package and our customer install_packages.sh was deleted.

### CODE REVIEW

**Comment by duftler** [comment]
Would it be better to backup all the contents of the directory (maybe to a `/backup` subdirectory)? Figure if somebody is modifying that script, they're potentially modifying the templates as well (and maybe adding credentials files and related artifacts).

File: rosco-web/pkg_scripts/preInstall.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# backup install_packages.sh
+if [ -f /opt/rosco/config/packer/install_packages.sh ]; then
+	cp /opt/rosco/config/packer/install_packages.sh /opt/rosco/config/packer/install_packages.sh.orig
```
Comment (duftler) [inline rosco-web/pkg_scripts/preInstall.sh]: Please use 2-space indentation.

**Comment by layered-esell** [comment]
cleaned up the formatting and now everything is backed up to /opt/rosco/config/packer/backup with a few safety things to hopefully make sure the script exits with success.

**Comment by duftler** [comment]
LGTM.
Thanks for raising the issue and for contributing the fix.

**Comment by duftler** [comment]
Closes https://github.com/spinnaker/spinnaker/issues/625.

**Pull request merged**